README.md

```diff
@@ -6,81 +6,74 @@ Note that over time, this job, and all the jobs called by it, will be migrated t
 
 # Che release process
 
-# Phase 0 - permissions
```

Comment:
are these no longer relevant? don't we still publish to https://quay.io/application/eclipse-che-operator-kubernetes/eclipse-che-preview-kubernetes?tab=releases ? last one was 5hrs ago so this seems still important to document in case you and I are both hit by a falling comet :)